src/exhortServices.ts

```diff
@@ -68,32 +54,21 @@ function stackAnalysisService(pathToManifest: string, options: object): Promise<
  * @param source The source for which the token is being validated.
  * @returns A promise resolving after validating the token.
  */
-async function tokenValidationService(options, source): Promise<string> {
+async function tokenValidationService(options: object, source: string): Promise<string | undefined> {
   try {
-
     // Get token validation status code
     const tokenValidationStatus = await exhort.validateToken(options);
 
-    if (
-      tokenValidationStatus === 200
-    ) {
+    if (tokenValidationStatus === 200) {
```

Comment:
I know this is only formatting but can you take the chance to refactor into a switch?

Reply:
Technically this isnt even used anywhere, but I dont want to delete any of the snyk stuff thats not commented out :smile: will format anyways